pkgs/development/tools/parsing/tree-sitter/grammar.nix

```diff
@@ -58,4 +65,9 @@ stdenv.mkDerivation ({
     fi
     runHook postInstall
   '';
+
+  meta = {
+    broken = isBroken;
```

Comment:
Might as well rename it to `broken` and `inherit` it here IMO.

Reply:
`isBroken` is always false (since it is generated by the update script), the name indicates that is flagged as broken.
I don't mind changing it to `broken`.

The reason behind this choice is to avoid updating all the JSON files without a new and unnecessary attribute.